label_studio/tests/test_block_objects.py

```diff
@@ -22,6 +20,6 @@ def test_export(
     assert Task.objects.filter(is_labeled=True).count() == 0
 
 
-def worker_change_stats(time_seconds, tasks):
+def worker_change_stats(tasks):
```

Comment:
Previously this thread was passed the wrong arguments, and would never actually start. When it does, because of sqlite's locking behavior, sometimes it flakes out: https://github.com/heartexlabs/label-studio/actions/runs/5537143705/jobs/10105651923?pr=4511

I'm not sure that testing this code with two threads making the same call is optimal, in any case; in addition, the sleep() calls make this test run very slowly.

Reply:
Still not really sure what the goal of the thread in this test was, but I set it up such that it should be reliable on both sqlite and other DBs